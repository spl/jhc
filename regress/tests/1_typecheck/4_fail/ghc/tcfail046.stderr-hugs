ERROR "tcfail046.hs":9 - An instance of Eq (Process a) is required to derive Eq (Continuation a)
