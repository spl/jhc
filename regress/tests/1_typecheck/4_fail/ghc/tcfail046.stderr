
tcfail046.hs:10:50:
    No instance for (Eq (Process a))
      arising from the 'deriving' clause of a data type declaration
    Possible fix:
      add an instance declaration for (Eq (Process a))
      or use a standalone 'deriving instance' declaration,
           so you can specify the instance context yourself
    When deriving the instance for (Eq (Continuation a))

tcfail046.hs:22:25:
    No instance for (Eq (Process a))
      arising from the 'deriving' clause of a data type declaration
    Possible fix:
      add an instance declaration for (Eq (Process a))
      or use a standalone 'deriving instance' declaration,
           so you can specify the instance context yourself
    When deriving the instance for (Eq (Message a))
